### PR TITLE
[shopsys] renamed js files

### PR DESCRIPTION
**Pull request by pesektomas**
| Q             | A
| ------------- | ---
|Description, reason for the PR| We refactored all javascripts in #1545 but we kept the original file names. File names are unified with the class (or function) name in this PR
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| Yes 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/framework/assets/js/admin/components/FileItem.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { forceValidateElement } from '../validation/customization/validation';
+import { forceValidateElement } from '../validation/customization/validationInit';
```
Comment (s3tezsky) [inline packages/framework/assets/js/admin/components/FileItem.js]: Why is this changed? or why is not the `validation.js` removed at all. These files are the same except path in `import` part.

File: packages/framework/assets/js/admin/components/FileUpload.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import './jquery.dmuploader';
-import FileItem from './fileUpload.fileItem';
-import Window from '../utils/window';
-import { forceValidateElement } from '../validation/customization/validation';
-import formChangeInfo from '../components/formChangeInfo';
-import Ajax from '../../common/utils/ajax';
-import Register from '../../common/utils/register';
+import FileItem from './FileItem';
+import Window from '../utils/Window';
+import { forceValidateElement } from '../validation/customization/validationInit';
```
Comment (s3tezsky) [inline packages/framework/assets/js/admin/components/FileUpload.js]: Same question as the one above.

File: project-base/assets/js/frontend/cart/index.js [hunk 3]
```diff
@@ -1,3 +1,3 @@
-import './cartBox';
-import './cartRecalculator';
-import './cartBoxItemRemover.js';
+import './CartBox';
+import './CartRecalculator';
+import './CartBoxItemRemover.js';
```
Comment (s3tezsky) [inline project-base/assets/js/frontend/cart/index.js]: Just a question - why is here the `.js` suffix?

File: project-base/assets/js/frontend/components/lazyLoadInit.js [hunk 4]
```diff
@@ -8,10 +8,10 @@ new MiniLazyload({
     placeholder: '/assets/frontend/images/noimage.png'
 }, '', MiniLazyload.IGNORE_NATIVE_LAZYLOAD);
 
-export function lazyLoadCall (container) {
```
Comment (s3tezsky) [inline project-base/assets/js/frontend/components/lazyLoadInit.js]: The old usage is still present in `upgrade/upgrade-instruction-for-webpack.md` and `doc/frontend/lazyload.md`

File: packages/framework/assets/js/admin/components/FileItem.js (hunk 1 above)
Comment (pesektomas) [reply packages/framework/assets/js/admin/components/FileItem.js]: 374c67a886d912b7eaaaeec57089a93414dca4d8

File: packages/framework/assets/js/admin/components/FileUpload.js (hunk 2 above)
Comment (pesektomas) [reply packages/framework/assets/js/admin/components/FileUpload.js]: 374c67a886d912b7eaaaeec57089a93414dca4d8

File: project-base/assets/js/frontend/cart/index.js (hunk 3 above)
Comment (pesektomas) [reply project-base/assets/js/frontend/cart/index.js]: 374c67a886d912b7eaaaeec57089a93414dca4d8

File: project-base/assets/js/frontend/components/lazyLoadInit.js (hunk 4 above)
Comment (pesektomas) [reply project-base/assets/js/frontend/components/lazyLoadInit.js]: 3e23cf9e30906af7b6772be85608e1ec14c0ea24
